### DISCUSSION TITLE
USDZ Models Don't Show Up When Selected Through a Div

### DISCUSSION THREAD

**Discussion by ahmetbugrayasarnara** [google/model-viewer#2360] [Q&A]
### Description

I have a product similiar to the example you put for dimensions. (=> [https://modelviewer.dev/examples/annotations/index.html#dimensions](https://modelviewer.dev/examples/annotations/index.html#dimensions))
However, I can't get it to work for iOS. I've written a function to detect user OS to change the label and select id like so;

`<label for="<?php echo $id; ?>">Product:</label>
        <select id="<?php echo $id; ?>">`

So that the users will get .glb files when on Android, and will get .usdz files on iOS. It currently only doesn't work in iPhones.
I've checked the files on the list, they are .usdz files. I get the loading bar for each model, but don't get the model on the screen and I don't get the AR button.

#### Live Demo

http://www.ahmetbugrayasar.com.tr/webxr/Furniture/

### Version

- model-viewer: v1.6.0

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [x] Safari 14 on iOS 14.4

### OS

- [ ] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR
- [ ] WebXR
- [x] SceneViewer
- [x] QuickLook

Thanks in advance for your help!

**Comment by elalish** [comment]
model-viewer only accepts glTF/GLB as a `src`, on all browsers. The only place you can put a USDZ file is `ios-src`, which is only used to open QuickLook for AR.
